source/positions/edu_uiowa_jobs_70717.md

```diff
@@ -0,0 +1,24 @@
+---
+position:
+  source_url: https://jobs.uiowa.edu/pands/view/70717
+  type: Full Time
+  title: PHOENIX APPLICATION DEVELOPER
+  company: University of Iowa
+  company_url: uiowa.edu
```

Comment:
Thanks for the PR, Chris! The build is failing because this needs `http://` (without that, it would be a broken relative link according to `linkchecker`).  I'm not at a computer right now, but I can take care of this if you don't have a chance to do so.